ui/components-react/src/test/filter-builder/FilterBuilderRuleGroup.test.tsx

```diff
@@ -0,0 +1,197 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
I think borderless buttons that are colored `--iui-icons-color` when not hovered and with 8px gap is the way to go:

![image](https://user-images.githubusercontent.com/70327485/175945718-b0132e2b-33a1-41e2-8ebc-61459dc16b8e.png)

Reply:
Looks good. Made changes in [1125ad5](https://github.com/iTwin/itwinjs-core/pull/3655/commits/1125ad52175e373d5fe71633a3c2f1631519efea)